kalite/settings/base.py

```diff
@@ -352,6 +413,7 @@
     'securesync.middleware.DBCheck',
     'django.middleware.common.CommonMiddleware',
     'kalite.distributed.middleware.LockdownCheck',
+    'kalite.distributed.middleware.LogRequests',
```

Comment:
This one did the trick of logging requests and response status codes which have been gravely missing from people's log submissions.